### PR TITLE
JF-7 Navbar

### PR DESCRIPTION
**Pull request by codyrae18**
## [JF-7: Navbar](https://roguereactdevs.atlassian.net/browse/JF-7)

**Description of Changes**
- Added Jedi Logo
- Added Styled Components
- Added 3 Link tab sections
     * Component 
     * Layouts
     * Style Components 

Image down below ---
image

### CODE REVIEW

**Review by hemeshvpatel — COMMENTED** [review]
Is this your latest changes? ( Seems last commit is 18 days old )

File: frontend/src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -8,23 +9,81 @@ interface INavBar {
...
+export const NavBar: React.FC<INavBar> = ({}
+) => {
     return (
         <OuterDiv>
-            This is my navbar
+            <Container>
+                <Logo>Jedi</Logo>
+                <Menu>
+                    <Item>
+                        <Link>Components</Link>
```
Comment (hemeshvpatel) [inline frontend/src/components/NavBar/NavBar.tsx:76]: Would make this an array up top or refactor into passed in props with something like title, url

**Comment by codyrae18** [comment]
So I just realize last night i forgot to push some stuff, i decided to do it and see if there's some changes want to do here before I'll say work complete

File: frontend/src/components/NavBar/NavBar.tsx (hunk 1 above)
Comment (codyrae18) [reply frontend/src/components/NavBar/NavBar.tsx:76]: Sure, i can do that

**Review by ablanco12 — APPROVED** [review]

File: frontend/src/components/NavBar/NavBar.tsx (hunk 1 above)
Comment (codyrae18) [reply frontend/src/components/NavBar/NavBar.tsx]: Task complete :)

**Comment by hemeshvpatel** [comment]
Looks like you have some conflicts on the PR

**Comment by codyrae18** [comment]
I fix the merge conflicts, Hem. Thanks bud

**Review by hemeshvpatel — APPROVED** [review]

**Pull request merged**